data/i18n/en/en.toml

```diff
@@ -35,6 +30,12 @@ other = "Twitter"
 [community_youtube_name]
 other = "YouTube"
 
+# Avoid using conjunction_1.
+# Must match the context in layouts/shortcodes/release-data.html
+# Appears on https://kubernetes.io/releases/
+# For example the "and" in "Complete 1.25 Schedule and Changelog"
```

Comment:
Not sure if it's good practice to limit the use of a string with general name.

Reply:
The conjunction is useful in a specific context. However, if it is used in many contexts, not all those contexts might localize the same way.